src/Compilers/Core/Portable/InternalUtilities/StringTable.cs

```diff
@@ -111,10 +111,9 @@ public void Free()
 
         #endregion // Poolable
 
-        internal string Add(char[] chars, int start, int len)
+        internal string Add(ReadOnlySpan<char> chars)
         {
-            var span = chars.AsSpan(start, len);
```

Comment:
note that these layers were also often using spans.  Now they don't use spans partially for some op, and array+start+length for others.  They use spans consistently through teh entire path.